api/docs/docs/updates.md

```diff
@@ -7,6 +7,27 @@ description: Updates to the Covid Act Now API.
 
 Updates to the API will be reflected here.
 
+### CDC Community Level data now available 
+_Added on 2022-04-05_
+
+The CDC Community Level metric as well as the subcomponents of the CDC Community Level metric are now available within the Covid Act Now API. 
+
+[Learn more about how the CDC Community Level is measured](https://covidactnow.org/covid-risk-levels-metrics#how-covid-risk-is-determined). 
+
+Fields added:
+* `communityLevels.cdcCommunityLevel`: The raw CDC Community Level metric. It serves as an exact mirror of the CDC’s published data, which is available for counties only and typically updates once a week.
+* `communityLevels.canCommunityLevel`: The Covid Act Now team’s version of CDC Community Level metric. Unlike the raw CDC community level, it is available for states and metros in addition to counties. It also uses different data sources in some cases (New York Times for cases, HHS for state hospitalization data). It updates daily, though county hospitalization data only updates once a week.
```

Comment:
Maybe change "Unlike the raw CDC community level" to "It uses the same methodology as the CDC community level, but ..."